[CI] CorruptedFileIT.testReplicaCorruption failure

Failed for 7.0: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.0+intake/859/console

## Reproduction line
(does not) reproduce locally
```
./gradlew :server:integTest --tests "org.elasticsearch.index.store.CorruptedFileIT.testReplicaCorruption" \
  -Dtests.seed=56970DA58EAFB57B \
  -Dtests.security.manager=true \
  -Dtests.locale=sv \
  -Dtests.timezone=Africa/Casablanca \
  -Dcompiler.java=12 \
  -Druntime.java=8
```

## Example relevant log:
```
org.elasticsearch.index.store.CorruptedFileIT > testReplicaCorruption FAILED
    java.lang.AssertionError: timed out waiting for green state
        at org.junit.Assert.fail(Assert.java:88)
        at org.elasticsearch.test.ESIntegTestCase.ensureColor(ESIntegTestCase.java:980)
        at org.elasticsearch.test.ESIntegTestCase.ensureGreen(ESIntegTestCase.java:936)
        at org.elasticsearch.test.ESIntegTestCase.ensureGreen(ESIntegTestCase.java:925)
        at org.elasticsearch.index.store.CorruptedFileIT.testReplicaCorruption(CorruptedFileIT.java:602)

    java.lang.AssertionError: not all translog generations have been released

    com.carrotsearch.randomizedtesting.UncaughtExceptionError: Captured an uncaught exception in thread: Thread[id=1401, name=elasticsearch[node_td1][generic][T#2], state=RUNNABLE, group=TGRP-CorruptedFileIT]

        Caused by:
        java.lang.AssertionError: wtf? file=corrupted_bNYq4690The4_a_bS3D1eg
```
```
 1> org.apache.lucene.index.CorruptIndexException: checksum failed (hardware problem?) : expected=r76dkc actual=17kuudq (resource=name [_0.cfs], length [11566], checksum [r76dkc], writtenBy [8.0.0]) (resource=VerifyingIndexOutput(_0.cfs))
  1> 	at org.elasticsearch.index.store.Store$LuceneVerifyingIndexOutput.readAndCompareChecksum(Store.java:1237) ~[main/:?]
  1> 	at org.elasticsearch.index.store.Store$LuceneVerifyingIndexOutput.writeByte(Store.java:1215) ~[main/:?]
  1> 	at org.elasticsearch.index.store.Store$LuceneVerifyingIndexOutput.writeBytes(Store.java:1245) ~[main/:?]
  1> 	at org.elasticsearch.indices.recovery.MultiFileWriter.innerWriteFileChunk(MultiFileWriter.java:120) ~[main/:?]
  1> 	at org.elasticsearch.indices.recovery.MultiFileWriter.access$000(MultiFileWriter.java:43) ~[main/:?]
  1> 	at org.elasticsearch.indices.recovery.MultiFileWriter$FileChunkWriter.writeChunk(MultiFileWriter.java:200) ~[main/:?]
  1> 	at org.elasticsearch.indices.recovery.MultiFileWriter.writeFileChunk(MultiFileWriter.java:68) ~[main/:?]
  1> 	at org.elasticsearch.indices.recovery.RecoveryTarget.writeFileChunk(RecoveryTarget.java:459) ~[main/:?]
  1> 	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$FileChunkTransportRequestHandler.messageReceived(PeerRecoveryTargetService.java:632) ~[main/:?]
  1> 	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$FileChunkTransportRequestHandler.messageReceived(PeerRecoveryTargetService.java:606) ~[main/:?]
  1> 	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:63) [main/:?]
  1> 	at org.elasticsearch.transport.TcpTransport$RequestHandler.doRun(TcpTransport.java:1077) [main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:751) [main/:?]
  1> 	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [main/:?]
  1> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_212]
  1> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_212]
  1> 	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```


## Frequency
Not often, last occurrence was April 3rd 2019.

--- a/server/src/test/java/org/elasticsearch/index/store/CorruptedFileIT.java
+++ b/server/src/test/java/org/elasticsearch/index/store/CorruptedFileIT.java
@@ -53,6 +53,7 @@ import org.elasticsearch.common.lucene.Lucene;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.common.unit.ByteSizeUnit;
 import org.elasticsearch.common.unit.ByteSizeValue;
+import org.elasticsearch.common.unit.TimeValue;
 import org.elasticsearch.env.NodeEnvironment;
 import org.elasticsearch.index.Index;
 import org.elasticsearch.index.IndexSettings;
@@ -606,7 +607,7 @@ public class CorruptedFileIT extends ESIntegTestCase {
             Settings.builder().putNull(EnableAllocationDecider.CLUSTER_ROUTING_ALLOCATION_ENABLE_SETTING.getKey())
         ));
 
-        ensureGreen();
+        ensureGreen(TimeValue.timeValueSeconds(60));
     }
 
     private int numShards(String... index) {

